Cannot compile to NodeJS

This line in the README fails at my system:

```
lein cljsbuild once; chmod +x ./target/org-parser.js
```

```
$ java -version
openjdk version "1.8.0_292"
OpenJDK Runtime Environment (build 1.8.0_292-b10)
OpenJDK 64-Bit Server VM (build 25.292-b10, mixed mode)
$ lein --version
Leiningen 2.9.5 on Java 1.8.0_292 OpenJDK 64-Bit Server VM
$ clojure --version
Clojure CLI version 1.10.3.839
```

`lein cljsbuild once` outputs:

```
Compiling ClojureScript...
Compiling ["out/test_cljs_with_node.js"] from ["src" "test"]...
Compiling ["out/test_cljs_with_node.js"] failed.
clojure.lang.ExceptionInfo: failed compiling file:test/org_parser/test_runner.cljs {:file #object[java.io.File 0x1dde5b0a "test/org_parser/test_runner.cljs"], :clojure.error/phase :compilation}
	at cljs.compiler$compile_file$fn__3901.invoke(compiler.cljc:1706)
	at cljs.compiler$compile_file.invokeStatic(compiler.cljc:1666)
	at cljs.compiler$compile_file.invoke(compiler.cljc:1642)
	at cljs.closure$compile_file.invokeStatic(closure.clj:653)
	at cljs.closure$compile_file.invoke(closure.clj:631)
	at cljs.closure$fn__5388.invokeStatic(closure.clj:727)
	at cljs.closure$fn__5388.invoke(closure.clj:721)
	at cljs.closure$fn__5301$G__5294__5308.invoke(closure.clj:549)
	at cljs.closure$compile_sources$iter__5514__5518$fn__5519.invoke(closure.clj:1088)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.next(RT.java:709)
	at clojure.core$next__5371.invokeStatic(core.clj:64)
	at clojure.core$dorun.invokeStatic(core.clj:3142)
	at clojure.core$doall.invokeStatic(core.clj:3148)
	at clojure.core$doall.invoke(core.clj:3148)
	at cljs.closure$compile_sources.invokeStatic(closure.clj:1084)
	at cljs.closure$compile_sources.invoke(closure.clj:1073)
	at cljs.closure$build.invokeStatic(closure.clj:3011)
	at cljs.closure$build.invoke(closure.clj:2919)
	at cljs.build.api$build.invokeStatic(api.clj:208)
	at cljs.build.api$build.invoke(api.clj:189)
	at cljs.build.api$build.invokeStatic(api.clj:195)
	at cljs.build.api$build.invoke(api.clj:189)
	at cljsbuild.compiler$compile_cljs$fn__909.invoke(compiler.clj:61)
	at cljsbuild.compiler$compile_cljs.invokeStatic(compiler.clj:60)
	at cljsbuild.compiler$compile_cljs.invoke(compiler.clj:48)
	at cljsbuild.compiler$run_compiler.invokeStatic(compiler.clj:168)
	at cljsbuild.compiler$run_compiler.invoke(compiler.clj:129)
	at user$eval1038$iter__1086__1090$fn__1091$fn__1117.invoke(form-init7231116708684083282.clj:1)
	at user$eval1038$iter__1086__1090$fn__1091.invoke(form-init7231116708684083282.clj:1)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.next(RT.java:709)
	at clojure.core$next__5371.invokeStatic(core.clj:64)
	at clojure.core$dorun.invokeStatic(core.clj:3142)
	at clojure.core$doall.invokeStatic(core.clj:3148)
	at clojure.core$doall.invoke(core.clj:3148)
	at user$eval1038.invokeStatic(form-init7231116708684083282.clj:1)
	at user$eval1038.invoke(form-init7231116708684083282.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7166)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	at clojure.lang.Compiler.loadFile(Compiler.java:7573)
	at clojure.main$load_script.invokeStatic(main.clj:452)
	at clojure.main$init_opt.invokeStatic(main.clj:454)
	at clojure.main$init_opt.invoke(main.clj:454)
	at clojure.main$initialize.invokeStatic(main.clj:485)
	at clojure.main$null_opt.invokeStatic(main.clj:519)
	at clojure.main$null_opt.invoke(main.clj:516)
	at clojure.main$main.invokeStatic(main.clj:598)
	at clojure.main$main.doInvoke(main.clj:561)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.main.main(main.java:37)
Caused by: clojure.lang.ExceptionInfo: null #:clojure.error{:source nil, :line nil, :column nil, :phase :compilation}
	at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:4128)
	at cljs.analyzer$analyze.invoke(analyzer.cljc:4114)
	at cljs.compiler$emit_source.invokeStatic(compiler.cljc:1524)
	at cljs.compiler$emit_source.invoke(compiler.cljc:1497)
	at cljs.compiler$compile_file_STAR_$fn__3870.invoke(compiler.cljc:1609)
	at cljs.compiler$with_core_cljs.invokeStatic(compiler.cljc:1417)
	at cljs.compiler$with_core_cljs.invoke(compiler.cljc:1406)
	at cljs.compiler$compile_file_STAR_.invokeStatic(compiler.cljc:1593)
	at cljs.compiler$compile_file_STAR_.invoke(compiler.cljc:1586)
	at cljs.compiler$compile_file$fn__3901.invoke(compiler.cljc:1691)
	... 56 more
Caused by: clojure.lang.ExceptionInfo: No such namespace: doo.runner, could not locate doo/runner.cljs, doo/runner.cljc, or JavaScript source providing "doo.runner" in file test/org_parser/test_runner.cljs {:tag :cljs/analysis-error}
	at cljs.analyzer$error.invokeStatic(analyzer.cljc:751)
	at cljs.analyzer$error.invoke(analyzer.cljc:747)
	at cljs.analyzer$error.invokeStatic(analyzer.cljc:749)
	at cljs.analyzer$error.invoke(analyzer.cljc:747)
	at cljs.analyzer$analyze_deps.invokeStatic(analyzer.cljc:2567)
	at cljs.analyzer$analyze_deps.invoke(analyzer.cljc:2541)
	at cljs.analyzer$ns_side_effects.invokeStatic(analyzer.cljc:3993)
	at cljs.analyzer$ns_side_effects.invoke(analyzer.cljc:3988)
	at cljs.analyzer$analyze_STAR_$fn__2657.invoke(analyzer.cljc:4112)
	at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
	at clojure.core$reduce.invokeStatic(core.clj:6827)
	at clojure.core$reduce.invoke(core.clj:6810)
	at cljs.analyzer$analyze_STAR_.invokeStatic(analyzer.cljc:4112)
	at cljs.analyzer$analyze_STAR_.invoke(analyzer.cljc:4102)
	at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:4131)
	... 65 more
Subprocess failed (exit code: 1)
```

--- a/README.org
+++ b/README.org
@@ -154,13 +154,13 @@ possible.
 
 #+RESULTS:
 : {:headlines [{:headline {:level 1, :title "Header with repeater"}, :section {:ast [[:text [:text-normal "  SCHEDULED: "] [:text-normal "<2019-11-27 Wed "] [:text-normal "+1d>"]]], :raw ["  SCHEDULED: <2019-11-27 Wed +1d>"]}}]}
-: 
+:
 
 ** NodeJS
 
 First, compile =org-parser= with:
 
-: lein cljsbuild once; chmod +x ./target/org-parser.js
+: lein cljsbuild once main; chmod +x ./target/org-parser.js
 
 =./target/org-parser.js file.org=, for example:
 
@@ -183,7 +183,7 @@ First, compile =org-parser= with: =lein uberjar=.
 
 #+RESULTS:
 : {:headlines [{:headline {:level 1, :title "Header with repeater"}, :section {:ast [[:text [:text-normal "  SCHEDULED: "] [:text-normal "<2019-11-27 Wed "] [:text-normal "+1d>"]]], :raw ["  SCHEDULED: <2019-11-27 Wed +1d>"]}}]}
-: 
+:
 
 Note: The =*= character must be replaced with the current version
 number of org-parser.

--- a/project.clj
+++ b/project.clj
@@ -16,7 +16,7 @@
   :profiles {:uberjar {:aot :all}}
   :doo {:build "test-cljs-with-node"}
   :cljsbuild
-  {:builds [{:id "foo"
+  {:builds [{:id "main"
              :source-paths ["src"]
              :compiler {:optimizations :none
                         :target :nodejs

